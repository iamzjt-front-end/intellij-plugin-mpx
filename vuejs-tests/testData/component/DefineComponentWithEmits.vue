<script>
import {defineComponent} from "vue"

export default defineComponent(
  {
    name: "TestComponent",
    emits: ["test-event"]
  })
</script>