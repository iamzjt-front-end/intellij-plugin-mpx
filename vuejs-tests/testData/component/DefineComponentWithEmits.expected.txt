{
 class: "VueSourceComponent",
 defaultName: "TestComponent",
 emits: [
  {
   class: "VueSourceEmitDefinition",
   name: "test-event",
  },
 ],
}
